morango/__init__.py

```diff
@@ -3,4 +3,4 @@
 from __future__ import unicode_literals
 
 default_app_config = "morango.apps.MorangoConfig"
-__version__ = "0.6.6"
+__version__ = "0.6.6-beta"
```

Comment:
Do we need to use PeP440 in order for PyPI to handle this properly?

Something like `0.6.6b0`?

https://www.python.org/dev/peps/pep-0440/#pre-releases

Reply:
Ah yes that's what we want. I looked at Kolibri and saw `-alpha` on the releases so went with that

Reply:
Yes, Kolibri turns the version tuple into a PEP compliant version string.